lib/ansible/modules/net_tools/networkd.py

```diff
@@ -0,0 +1,650 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2018, Susant Sahani
+# Copyright: (c) 2018, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)#
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: networkd
+short_description: Configure Link, NetDev and Network for systemd-networkd
+description:
+    - Configure Link, NetDev and Network for systemd-networkd
+
+options:
+  ConfigType:
```

Comment:
The capitalization casing doesn't match what is typical in Ansible. We heavily prefer `snake_case` to `CamelCase` for module parameters, so mixing up snake/camel options isn't a source of frustration for users.